src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -4155,6 +4398,23 @@ namespace AirflowNetworkBalanceManager {
 		static bool OneTimeFlag( true );
 		static bool ErrorsFound( false );
 		Real64 GlobalOpenFactor;
+		Real64 ZonePressure1;
```

Comment:
Since it's all if-else logic here, I'm not sure you'll be able to move them into a tighter scope easily.  Is CI complaining about possibly uninitialized variables @mjwitte?  If not, I think these might be OK.

_That said_, it really doesn't hurt to give them an initialized value when you define them here, either.  Something that will cause trouble if you use it "uninitialized" later....like negative 100000....